ios/Handlers/ResourcesHandler.swift

```diff
@@ -1,12 +1,13 @@
+// swiftlint:disable identifier_name
```

Comment:
Correct me if I'm wrong but I think you need to cancel this eg at the end of file, otherwise it will not obey this rule for further check.

Reply:
You don't need to re-enable it at the end of the file, the disable only counts for that file. https://academy.realm.io/posts/slug-jp-simard-swiftlint/